src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -107,35 +110,48 @@ export default Backbone.Model.extend({
       [dateType]: this.formatDateRange(relativeRange, unit, dateFormat),
     };
   },
+  getDefaultStatesFilter() {
+    const isDoneOnly = this.id === 'done-last-thirty-days';
+    const { done, notDone } = this.states.groupByDone();
+
+    return isDoneOnly ? done : notDone;
+  },
+  setDefaultFilterStates() {
+    const defaultStatesFilter = this.getDefaultStatesFilter();
+    this.set({ filters: { states: defaultStatesFilter.map('id'), worklistId: this.id } });
```

Comment:
I added the `worklistId: this.id` here. To change the filters state into this:

```js
filters: {
  insurance: 'Medicare',
  states: ['id', 'id'],
  worklistId: 'owned-by'
}
```

Because the `filters_state.js` doesn't have access to the `state.id` worklist id.

Hoping for a better way to do this. Since I don't like that the `state.id` and `state.filters.worklistId` are duplicates.